src/components/my-history/custom-my-history.vue

```diff
@@ -22,6 +22,16 @@
 
             <div class="x-flex x-flex-col x-gap-2">
               <p>{{ suggestion.query }}</p>
+              <ul v-if="suggestion.selectedFilters" class="x-flex x-gap-8">
```

Comment:
The `v-if="suggestion.selectedFilters"` condition is passing even when there are no filters. An empty array returns `true` when converted to boolean. Use `length`.